src/content/self-hosted/install/releases.mdx

```diff
@@ -5,6 +5,21 @@ sidebar_label: Release Notes
 id: releases
 ---
 
+## 1.3.0
+
+### New Feature
+
+- Git Catalog feature will be part of the core of Okteto
```

Comment:
```suggestion
- Git Catalog to provide predefined development environments to every developer
```

Reply:
Can we add an item about overwriting the autoscaler and init container images, and link the docs from here. This is for the case of companies that cannot pull from dockerhub. They can push `busybox` to their private registry, and overwrite these images

Reply:
I would also mention https://github.com/okteto/app/pull/5045 and https://github.com/okteto/app/pull/5044 as features

Reply:
@pchico83 I found strange that I couldn't find the prs related to the autoscaler and init container in the changelog. Turns out they have not made it to the release 1.3

Here are the 2 prs:
https://github.com/okteto/app/pull/5224
https://github.com/okteto/app/pull/5223

Should we backport them?

Reply:
@AgustinRamiroDiaz no, let's not add them to list of features then

Reply:
@pchico83 I'll remove the configuration associated to that too